app/components/subheader/schema.yml

```
@@ -1,10 +1,33 @@
 _description: |
   A simple subheader component.
 
+type:
+  _label: Type
+  _has:
+    input: radio
+    options:
+      - name: "small (h4)"
+        value: "h4"
+      - name: "medium (h3)"
+        value: "h3"
+      - name: "large (h2)"
+        value: "h2"
 text:
   _label: Subheader Text
-  _placeholder:
-    height: 20px
-    text: Subheader Text
   _has:
-    input: inline
+    input: text
+subheaderid:
```

Please use camelCase
```suggestion
subheaderId:
```